src/System.Console/src/System/ConsolePal.Unix.cs

```diff
@@ -1102,18 +1186,21 @@ internal static unsafe int Read(SafeFileHandle fd, byte[] buffer, int offset, in
         /// <param name="buffer">The buffer from which to write data.</param>
         /// <param name="offset">The offset at which the data to write starts in the buffer.</param>
         /// <param name="count">The number of bytes to write.</param>
-        private static unsafe void Write(SafeFileHandle fd, byte[] buffer, int offset, int count)
+        /// <param name="noCursorChange">Writing this buffer doesn't change the cursor position.</param>
+        private static unsafe void Write(SafeFileHandle fd, byte[] buffer, int offset, int count, bool noCursorChange = false)
```

Comment:
Could we flip the polarity of the new argument to instead be `bool mayChangeCursorPosition = true`?